scp/cc/cpio/client_providers/kms_client_provider/src/BUILD.bazel

```diff
@@ -27,13 +28,20 @@ cc_library(
                 "//scp/cc/public/cpio/interface:aws_cpio_lib_outside_tee": [
                     "//scp/cc/cpio/client_providers/kms_client_provider/src/aws:nontee_aws_kms_client_provider_lib",
                 ],
+                # We should split implementation here. Currently it uses fake attestation silently when it's outside TEE.
+                "//scp/cc/public/cpio/interface:azure_cpio_lib_inside_tee": [
+                    "//scp/cc/cpio/client_providers/kms_client_provider/src/azure:azure_kms_client_provider_lib",
+                ],
+                "//scp/cc/public/cpio/interface:azure_cpio_lib_outside_tee": [
+                    "//scp/cc/cpio/client_providers/kms_client_provider/src/azure:azure_kms_client_provider_lib",
+                ],
```

Comment:
We will separate these two properly in a future PR. Current `azure_cpio_lib_inside_tee` uses fake attestation report silently when it's outside TEE.